dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/utils/DefaultAnalyticsTestUtils.java

```diff
@@ -0,0 +1,113 @@
+package org.hisp.dhis.analytics.utils;
```

Comment:
This class has no dependencies. Remove interface and convert to AnalyticsTestUtils class with static methods.